### ISSUE TITLE
poetry test failure

### ISSUE THREAD

**Issue by o-smirnov** [caracal-pipeline/stimela2#75]
@SpheMakh, the ``import tests.stimela_tests`` way is not playing nicely again:
```
$ poetry install --with tests
$ pytest tests/  # this runs OK
$ pip install -e ~/projects/CubiCal/
$ pytest tests/ # this now fails
```

I think this is because CubiCal may have its own tests package that ends up overwriting stimela's tests. Unfortunately this means the testing is very easy to break in test mode. I think we need to go back to importing stimela_tests from root level somehow (back to ugly symlinks?)

**Comment by o-smirnov** [comment]
@sjperkins your input might be needed too.

**Comment by sjperkins** [comment]
Try `$ poetry run pytest tests`

**Comment by sjperkins** [comment]
Poetry automatically manages a virtual environment for the project that is loaded when

- `poetry run`
- `poetry shell`

are executed.

**Comment by sphemakh** [comment]
Try again using branch `issue-75`

**Comment by sphemakh** [comment]
Fixed in #76
